lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py

```diff
@@ -312,7 +312,7 @@ def test_basic_normal_mean_predictor(self):
             predictor_of_mean_flag, distribution)
         self.assertIsInstance(result, np.ndarray)
         self.assertArrayAlmostEqual(
-            result, [-0.08169791, -0.09784413, 0.00822535, 1.00956199])
+            result, [-0.059093, -0.099905, 0.008257, 1.009563])
```

Comment:
I've had a look at these values, and I think that the differences are similar in explanation to the Test_compute_initial_guess.test_mean_predictor_estimate_coefficients_nans test, as the inputs to the CRPS minimisation are only different at at a high level of precision e.g. for the forecast_predictor_data input to the minimisation:
```
229.4833221435547 240.7333221435547 251.9833221435547 263.2333068847656
229.48333740234375 240.73333740234375 251.98333740234375 263.23333740234375
```
The instability in the points computed by the minimisation is likely caused by the fact that the values in these unit tests are idealised, and there isn't that many values, so the minimisation result can move around quite easily.